app/controllers/api/v1/subject_sets_controller.rb

```diff
@@ -86,7 +86,6 @@ def destroy_relation(resource, relation, value)
   private
 
   def remove_linked_set_member_subjects(resource, set_member_subjects)
-    SubjectWorkflowCount.where(set_member_subject: set_member_subjects).delete_all
```

Comment:
workflows yes - subjects sort of(https://github.com/zooniverse/Panoptes/issues/1193), no need for pg cascades we can just hook into the API lifecycle events like this method used to. If we go in and hack on the console and break things then that's our fault.